app/controllers/BaseController.java

```diff
@@ -49,19 +49,15 @@
 
     private static ObjectMapper mapper = BridgeObjectMapper.get();
 
-    protected AuthenticationService authenticationService;
-    protected StudyService studyService;
-    protected CacheProvider cacheProvider;
-    protected BridgeConfig bridgeConfig;
+    private BridgeConfig bridgeConfig;
+    private CacheProvider cacheProvider;
 
-    @Autowired
-    public void setStudyService(StudyService studyService) {
-        this.studyService = studyService;
-    }
+    StudyService studyService;
```

Comment:
I agree with Alx here. We shouldn't make this package-scoped. The tests can use the setters if they need access to these, and if subclasses need access, we should make them protected.